models/facility_test.go

```diff
@@ -19,9 +19,46 @@ func TestInsertFacility(t *testing.T) {
 		panic(err)
 	}
 
+	equals(t, facility.Type, returnedType)
+}
+
+func TestFindFacility(t *testing.T) {
+	ids, err := insertTestFacilities()
+	checkErr(err)
+	defer tearDown("facilities", "id IN ($1, $2)", ids[0], ids[1])
+	tx, err := tdb.Begin()
+	facility, err := tx.FindFacility("type = $1", "boathouse")
+	tx.Commit()
+	ok(t, err)
+
 	equals(t, "boathouse", facility.Type)
 }
 
 func newTestFacility() Facility {
 	return Facility{Type: "boathouse"}
 }
+
+func insertTestFacilities() (teardownIDs []int, err error) {
```

Comment:
I'm not fully happy with my solution for inserting information into the database. Maybe I should look into a factory_girl equivalent for Go, if such a thing exists.